RecoPixelVertexing/PixelTrackFitting/BuildFile.xml

```diff
@@ -9,24 +10,21 @@
 <use   name="FWCore/Framework"/>
 <use   name="FWCore/MessageLogger"/>
 <use   name="FWCore/ParameterSet"/>
-<use   name="FWCore/PluginManager"/>
 <use   name="FWCore/Utilities"/>
 <use   name="Geometry/CommonDetUnit"/>
 <use   name="Geometry/Records"/>
 <use   name="Geometry/TrackerGeometryBuilder"/>
 <use   name="MagneticField/Engine"/>
-<use   name="MagneticField/Records"/>
+<use   name="RecoTracker/TkHitPairs"/>
 <use   name="RecoTracker/TkMSParametrization"/>
 <use   name="RecoTracker/TkTrackingRegions"/>
 <use   name="TrackingTools/DetLayers"/>
-<use   name="TrackingTools/PatternTools"/>
 <use   name="TrackingTools/KalmanUpdators"/>
-<use   name="TrackingTools/Records"/>
+<use   name="TrackingTools/MaterialEffects"/>
```

Comment:
based on the comment in RecoPixelVertexing/PixelTrackFitting/src/KFBasedPixelFitter.cc, this should not be necessary